src/lib/components/PreFooter.svelte

```diff
@@ -126,6 +126,7 @@
                     </a>
                 </div>
             </li>
+            <!-- TODO: @itznotabug/@thejessewinton - the URL and the event name needs to be checked. -->
```

Comment:
this should be handled in here - https://github.com/appwrite/website/pull/1750. can be removed.